docs/content/en/snippets/week3.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Week 3 Snippet"
+date: 2022-02-11T23:42:55-08:00
+# The following are extra (optional) metadata for ananke
+# tags: ["tag1", "tag2"]
+# https://unsplash.com/photos/ETRPjvb0KM0, Photo by Patrick Perkins on Unsplash
+featured_image: "https://images.unsplash.com/photo-1503551723145-6c040742065b-v2"
+author: "Yueqian Zhang"
+description: "Good Start!"
+---
+
+<!-- For this first week snippet you should focus on any testing of project ideas as well as any work done for your proof of concept. -->
+
+We have got individual functions work this week!
+
+<!--more-->
+
+## New Features/Functionalities
+
+- brush prototype that can draw on a board.
+- 3D generation from buttons works!
+- File transfer is on its way.
+
+## Review Requests
+
+There's nothing needing review this week.
+
+## Blocking Issues
+
+There are no blocking issues this week and we don't need other assistance.
+
+## What's Next
+
+Next week, we'll refine our individual functions and pull everything together and hopefully and the "multiuser".
```

Comment:
```suggestion
Next week, we'll refine our individual functions and pull everything together, and hopefully the "multiuser".
```

Reply:
I have fixed this and the "Brush". I think it is fine to have week3 like this because this is before week4.

Reply:
I see. Please add the two PRs for review by staff with one sentence description for each, then this should be good to go